### PR TITLE
Fix vulnerability issues

### PR DESCRIPTION
**Pull request by arendarenko**
Fix for #438

### CODE REVIEW

File: Cron/Ecommerce.php [hunk 1]
```diff
@@ -128,8 +135,8 @@ public function execute()
                     $this->_apiResult->processResponses($storeId, true, $mailchimpStoreId);
                     $batchId = $this->_processStore($storeId, $mailchimpStoreId, $listId);
                     if ($batchId) {
-                        $connection->update($tableName, ['batch_id' => $batchId, 'mailchimp_sync_modified' => 0, 'mailchimp_sync_delta' => $this->_helper->getGmtDate()], "batch_id is null and mailchimp_store_id = '$mailchimpStoreId'");
-                        $connection->update($tableName, ['batch_id' => $batchId, 'mailchimp_sync_modified' => 0, 'mailchimp_sync_delta' => $this->_helper->getGmtDate()], "batch_id is null and mailchimp_store_id = '$listId'");
+                        $connection->update($tableName, ['batch_id' => $batchId, 'mailchimp_sync_modified' => 0, 'mailchimp_sync_delta' => $this->_helper->getGmtDate()], "batch_id is null and mailchimp_store_id = '".$this->escaper->escapeHtml($mailchimpStoreId)."'");
```
Comment (ihor-sviziev) [inline Cron/Ecommerce.php]: Escaping html isn't proper fix.

Looks like this code should be like this:

```suggestion
                        $connection->update(
                            $tableName,
                            ['batch_id' => $batchId, 'mailchimp_sync_modified' => 0, 'mailchimp_sync_delta' => $this->_helper->getGmtDate()],
                            ["batch_id is null and mailchimp_store_id = ?" => $mailchimpStoreId]
                        );
```

I took this example from 
https://github.com/magento/magento2/blob/b2ce2a37d921b5ad88fc38663fc0ff3dd6c582d1/app/code/Magento/Reports/Model/ResourceModel/Event.php#L67-L71

File: view/frontend/templates/mailchimpjs.phtml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <?php
-    $url = $this->getJsUrl();
+    $url = $block->escapeHtml($this->getJsUrl());
```
Comment (ihor-sviziev) [inline view/frontend/templates/mailchimpjs.phtml]: Actually escaping better to do just during printing variable, so it's better to add this code few lines below

**Comment by arendarenko** [comment]
Hi @gonzaloebiz @ihor-sviziev 
I did changes in code, please review them.

**Review by ihor-sviziev — APPROVED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
Hi @gonzaloebiz,
Could you review this PR? It looks good to me

**Review by centerax — APPROVED** [review]

**Comment by gonzaloebiz** [comment]
Look at #451

**Pull request closed without merging**